### PR TITLE
Remove old d3log

### PR DESCRIPTION
**Pull request by arkivm**
Remove old
* implementation of d3log
* test cases
* test cases from gitlab-CI

This PR is to make way for the new d3log implementation.

### CODE REVIEW

**Comment by mbudiu-vmw** [comment]
Can't we keep it around and put the new one on the side until it provides at least equivalent functionality?

**Comment by ryzhyk** [comment]
I support deleting it to reduce maintenance overheads. And I think the new version that will be merged soon is actually more functional.

File: test.sh [hunk 1]
```diff
@@ -325,36 +324,6 @@ souffle_tests7() {
 
 # 'd3log' test group.
 
-tcp_channel() {
```
Comment (ryzhyk) [inline test.sh:328]: You also want to remove stuff at line 107 (github doesn't let me annotate lines that haven't changed)

**Review by ryzhyk — APPROVED** [review]
Looks good, thanks!

**Comment by ryzhyk** [comment]
I'd also suggest merging the three commits in this PR into one.

File: test.sh (hunk 1 above)
Comment (arkivm) [reply test.sh:328]: fixed. thanks

**Pull request merged**